yamcs-core/src/main/java/org/yamcs/cfdp/FileDownloadRequests.java

```diff
@@ -0,0 +1,25 @@
+package org.yamcs.cfdp;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+/**
+ * Only mapping a transaction ID to a download bucket at the moment
+ */
+public class FileDownloadRequests {
+
+    private Map<CfdpTransactionId, String> buckets = new ConcurrentHashMap<>();
```

Comment:
Why use a concurrent hash map and make the methods synchronized at the same time?